AppController/djinn.rb

```diff
@@ -4695,9 +4713,12 @@ def setup_appengine_application(app, state)
 
       if @app_info_map[app]['nginx'].nil?
         @app_info_map[app]['nginx'] = nginx_app_port
+      end
+      if @app_info_map[app]['nginx_https'].nil?
+        @app_info_map[app]['nginx_https'] = nginx_app_https
```

Comment:
ah, understood. To be the best of knowledge that can happen only from restoring an old state (look for the variable @restored in start_appengine()), hence there shouldn't no be collision.